crates/topos-tce-storage/src/validator/tables.rs

```diff
@@ -78,8 +86,9 @@ impl ValidatorPerpetualTables {
             ColumnFamilyDescriptor::new(cfs::UNVERIFIED, default_options()),
         ];
 
-        let db = init_with_cfs(&path, default_options(), cfs)
-            .unwrap_or_else(|_| panic!("Cannot open DB at {:?}", path));
+        let db = init_with_cfs(&path, default_options(), cfs).unwrap_or_else(|e| {
+            panic!("Cannot open DB at {:?} => error {:?}", path, e);
```

Comment:
Do you think we should `error!("Cannot open DB at {:?} => error {:?}", path, e); exit(1)` here instead?

Reply:
Yes but not in this PR